docs/installation/webservers/apache.md

```diff
@@ -0,0 +1,197 @@
+![SeAT](http://i.imgur.com/aPPOxSK.png)
+
+# webserver - apache
+
+The SeAT web interface requires a web server to serve the HTML goodies it has. Apache is an alternative of nginx. You don't **have** to use it, so if you prefer something else, feel free.
+
+This guide contains two sections:
+
+* Docker setup
+* Host Setup (f.e. via SeAT-Tool or manual setup)
+
+## docker setup
+
+This part is about how to setup apache to serve your seat-docker installation:
+
+!!! info "Docker-Setup"
+
+    This part is still under construction.
+    
+A minimal Apache config might look like this
+
+````
+<VirtualHost *:80>
+    ServerName discourse.example.com
+    ProxyPass / http://0.0.0.0:8080/
+    ProxyPassReverse / http://0.0.0.0:8080/
+</VirtualHost>
+````
+
+---
+
+## host installations
+In order to get the SeAT fronted running, we need to configure Apache to serve our SeAT installs `public/` folder. This is the only folder that should be internet facing. That small `index.php` is the gateway into the application.
+The Apache configuration itself will depend on how your server is set up. Generally, virtual hosting is the way to go, and this is what this guide will be showing here.
+
+
+### Unbuntu Installation
+
+
+Let's install Apache & PHP7.2 mod:
+
+```
+sudo apt-get install apache2 libapache2-mod-php7.2
+```
+
+Duplicate the standard www configuration file:
+```
+sudo cp /etc/apache2/sites-available/000-default.conf /etc/apache2/sites-available/seat.conf
+``` 
+
+Next, update the configuration file at `/etc/apache2/sites-available/seat.conf` with some adequate values :
+
+```
+sudo vim /etc/apache2/sites-available/seat.conf
+```
+
+
+
+| variable | initial value | new value |
+| ------------ | ------------- | ------------ |
+| ServerAdmin | webmaster@localhost  | email@yourdomain.com |
+| DocumentRoot | /var/www/html  | /var/www/seat |
+| ServerName | -  | yourdomain.com |
+| ErrorLog | ${APACHE_LOG_DIR}/error.log  | ${APACHE_LOG_DIR}/seat-error.log |
+| CustomLog | ${APACHE_LOG_DIR}/access.log combined  | ${APACHE_LOG_DIR}/seat-access.log combined |
+
+Since SeAT is running under a different user other than Apache's `www-data` you might run into some permission issue. Hence using `mpm_itk` module can let Apache run as seat user to avoid permission problem. You don't **have** to use it.
```

Comment:
Please remove this. I prefer us using the normal operating system provided users such as `www-data`.

Reply:
i blindly copied this from a privous guide .... shall i completly delete Line 67?